openmc/deplete/chain.py

```diff
@@ -189,26 +190,38 @@ def __len__(self):
         return len(self.nuclides)
 
     @classmethod
-    def from_endf(cls, decay_files, fpy_files, neutron_files):
+    def from_endf(cls, decay_files, fpy_files, neutron_files, progress=True):
         """Create a depletion chain from ENDF files.
 
         Parameters
         ----------
-        decay_files : list of str
-            List of ENDF decay sub-library files
-        fpy_files : list of str
-            List of ENDF neutron-induced fission product yield sub-library files
-        neutron_files : list of str
-            List of ENDF neutron reaction sub-library files
+        decay_files : iterable of str or openmc.data.Decay
+            ENDF decay sub-library files or evalutated ``Decay`` instances
+        fpy_files : iterable of str or openmc.data.FissionProductYields
+            ENDF neutron-induced fission product yield sub-library files or
+            evaluated ``FissionProductYields``
+        neutron_files : iterable of str or openmc.data.endf.Evaluation
```

Comment:
To me, it seems a little weird to see this function take `Decay` and `FissionProductYields` for the other arguments but not `IncidentNeutron` for this one. How about just having all of them take either str or `Evaluation`?

Reply:
That's a good point. Since the [`Decay`](https://github.com/openmc-dev/openmc/blob/3f5373c71bd8331a18c61637214fb2372dcd47b3/openmc/data/decay.py#L314-L319) and [`FissionProductYields`](https://github.com/openmc-dev/openmc/blob/3f5373c71bd8331a18c61637214fb2372dcd47b3/openmc/data/decay.py#L132-L135) `__init__` methods already support passing a file path *or* an `Evaluation`, the intended feature for this PR is already implemented

Reply:
That's true. How would you like to proceed then? I think just clarifying the arguments in the docstring (that they accept `Evaluation` too) is sufficient.

Reply:
I'll update the docstrings, no problem. Thanks for the input on this